test/sql/copy/csv/test_date.test

```diff
@@ -37,4 +37,10 @@ query TT
 Select * from read_csv('data/csv/custom_date.csv', header=true, timestampformat='%m/%d/%Y, %-I:%-M %p');
 ----
 123	2023-01-02 12:33:00
-124	2023-12-02 11:57:00
\ No newline at end of file
+124	2023-12-02 11:57:00
+
+query TT
+Select * from read_csv('data/csv/custom_date.csv', header=true, timeformat='%m/%d/%Y, %-I:%-M %p', types = ['BIGINT', 'TIME'] );
+----
+123	12:33:00
+124	11:57:00
```

Comment:
I wish this test had some invalid data too, but the existing test does not so it is fine for now.

Reply:
Any tips on how to run just a subset of the unit tests? I'm afraid the docs aren't that forthcoming.

Reply:
Added a separate file for the failing tests.